README.md

```diff
@@ -99,7 +114,7 @@ To deploy this app follow the steps below:
     Note that this will likely spawn multiple processes as "nobody"
     depending on the number of server ports specified.
     
- #. The ``bin/django.wsgi`` script should make it 
+ 1. The ```bin/django.wsgi``` script should make it 
```

Comment:
There won't be any bin directory nor there would be django.wsgi.
There would be a wsgi.py which would be lying in the "online_test" project directory